Assets/_Laz/Scripts/Laz/LazCoordinatorBehaviour.cs

```diff
@@ -41,26 +42,28 @@ public void Initialize(LazPlayer lazPlayer, ILazoWrapped[] objectOfInterests)
             }
 
             lazPlayer.SetSpawn(transform.position);
-            lazPlayer.Set(_lazoScriptableObject, objectOfInterests);
-            lazPlayer.Set(_movementPropertyScriptableObject,lazPlayer.LazoTool);
+            lazPlayer.SetLazo(_lazoScriptableObject, objectOfInterests, _boostBehaviour);
```

Comment:
renaming it. And refactored to pass in IBoost implemented behaviour into the business logic instead of the monobehaviour